src/ngx_pagespeed.cc

```diff
@@ -1921,8 +1920,7 @@ ngx_int_t ps_resource_handler(ngx_http_request_t* r,
             cfg_s->server_context,
             options,
             ctx->base_fetch,
-            false /* requires_blink_cohort (no longer unused) */,
-            &page_callback_added);
+            false /* requires_blink_cohort (no longer unused) */);
```

Comment:
proxy_interface does pass true for the last argument if cache_html is on, so that argument isn't dead, at least.